bin/sl-deploy.txt

```diff
@@ -5,9 +5,17 @@ Deploy a node application to a StrongLoop process manager
 Options:
   -h,--help          Print this message and exit.
   -v,--version       Print version and exit.
-  -c,--config CFG    Deploy a specified configuration (default is "default").
+  -s,--service SVC   Deploy to service SVC.
 
 Arguments:
+  SVC: The name or ID of the service on the process manager.
+
+    Defaults to the name of the application specified in the package.json file.
+
+    If a service with the specified name is not available on the process
```

Comment:
also, if a service with the default name, it will be created?

I think this can be misread as only an explicitly speced SVC will be created, so should be clearer.